### PR TITLE
Display unique identifiers(Email Id) in add existing member functionality

### LINKED ISSUE #1738: Display unique identifiers(Email Id) in add existing member functionality
**Is your feature request related to a problem? Please describe.**
When adding an existing user to an organization in Talawa Admin, the interface currently displays only the names of the members. This can cause confusion when multiple members have similar or identical names.

**Describe the solution you'd like**
The system should display unique identifiers (such as email IDs) alongside member names to distinguish between members with similar names.

**Additional context**

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by krishna619**
**What kind of change does this PR introduce?**

feature - Displays unique identifiers(Email Id) in add existing member functionality

Fixes #1738 

**Did you add tests for your changes?**

No

**Snapshots/Videos:**

**If relevant, did you update the documentation?**

**Summary**

**Does this PR introduce a breaking change?**

**Other information**

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**

### CODE REVIEW

**Comment by krishna619** [comment]
@palisadoes please review.

**Comment by dhanagopu** [comment]
@krishna619 Please fix the failing tests

**Comment by dhanagopu** [comment]
@Kevoniat @beingnoble03 Please review this PR

**Comment by beingnoble03** [comment]
@krishna619 please add a screenshot

**Comment by krishna619** [comment]
@beingnoble03 Hi,
please find the SS 
image

**Comment by beingnoble03** [comment]
the above heading still says "User Name", can you make that right? 

maybe adding another column "Email" can fix it?

not sure if this would be the right way to do that? @aashimawadhwa what should be the right way to do that?

**Comment by beingnoble03** [comment]
@krishna619 I guess the simplest way would be to just replace "User Name" with "User", rest looks good.

**Comment by beingnoble03** [comment]
Add translations as well

**Comment by krishna619** [comment]
image
@beingnoble03 I have changed the column name and added translations as well.
If you and @aashimawadhwa feel like we need a separate column for email in future you can let me now i'll do the needful.

**Comment by krishna619** [comment]
@palisadoes please review.

File: src/screens/OrganizationPeople/AddMember.tsx [hunk 1]
```diff
@@ -347,9 +348,7 @@ function AddMember(): JSX.Element {
                   <TableHead>
                     <TableRow>
                       <StyledTableCell>#</StyledTableCell>
-                      <StyledTableCell align="center">
-                        User Name
-                      </StyledTableCell>
+                      <StyledTableCell align="center">User</StyledTableCell>
```
Comment (beingnoble03) [inline src/screens/OrganizationPeople/AddMember.tsx:351]: @krishna619 the `User` translation is still missing

**Review by beingnoble03 — CHANGES_REQUESTED** [review]

File: src/screens/OrganizationPeople/AddMember.tsx (hunk 1 above)
Comment (krishna619) [reply src/screens/OrganizationPeople/AddMember.tsx]: image
@beingnoble03 
added translation for user and add member , please review.

**Review by beingnoble03 — APPROVED** [review]
lgtm

**Review by palisadoes — APPROVED** [review]

**Pull request merged**